fhir-client/src/main/java/com/ibm/fhir/client/FHIRParameters.java

```diff
@@ -53,16 +59,17 @@ public String text() {
 
     /**
      * The valid set of value prefixes that can be used when constructing the parameters for a search operation.
+     * @implNote please keep in sync with @see SearchConstants.Prefix
```

Comment:
I don't think the `@see` tag will do anything special within the middle of a line like this.  Although it doesn't actually help much in Eclipse (since fhir-client has no dependency on fhir-search), I think it would be better to use @link as follows:
```suggestion
     * @implNote please keep in sync with {@link com.ibm.fhir.search.SearchConstants.Prefix}
```

Reply:
Updated.